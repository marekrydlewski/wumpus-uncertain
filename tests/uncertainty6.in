5 4
0.2
B???
OB??
OOB?
OB??
B???