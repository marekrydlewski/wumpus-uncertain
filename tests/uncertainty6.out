0.00 1.00 0.20 0.20 
0.00 0.00 0.41 0.20 
0.00 0.00 0.00 0.41 
0.00 0.00 0.41 0.20 
0.00 1.00 0.20 0.20 
